dts/common/nordic/nrf54h20_enga.dtsi

```diff
@@ -0,0 +1,310 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mem.h>
+#include <nordic/nrf_common.dtsi>
+
+#include <zephyr/dt-bindings/misc/nordic-nrf-ficr-nrf54h20-enga.h>
+
+/delete-node/ &sw_pwm;
+
+/ {
+	#address-cells = <1>;
+	#size-cells = <1>;
+
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpuapp: cpu@2 {
+			compatible = "arm,cortex-m33";
+			reg = <2>;
+			device_type = "cpu";
+			clock-frequency = <DT_FREQ_M(320)>;
+		};
+
+		cpurad: cpu@3 {
+			compatible = "arm,cortex-m33";
+			reg = <3>;
+			device_type = "cpu";
+			clock-frequency = <DT_FREQ_M(256)>;
+		};
+
+		cpuppr: cpu@d {
+			compatible = "nordic,vpr";
+			reg = <13>;
+			device_type = "cpu";
+			clock-frequency = <DT_FREQ_M(16)>;
+			riscv,isa = "rv32emc";
+			nordic,bus-width = <32>;
+		};
+	};
+
+	reserved-memory {
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		cpurad_uicr_ext: memory@e1ff000 {
+			reg = <0xe1ff000 DT_SIZE_K(2)>;
+		};
+
+		cpuapp_uicr_ext: memory@e1ff800 {
+			reg = <0xe1ff800 DT_SIZE_K(2)>;
+		};
+	};
+
+	clocks {
+		fll16m: fll16m {
+			compatible = "fixed-clock";
+			#clock-cells = <0>;
+			clock-frequency = <DT_FREQ_M(16)>;
+		};
+	};
```

Comment:
question for @57300, does PPR see FLL16M? If not maybe this node should be deleted in ppr dtsi

Reply:
> does PPR see FLL16M?

It's in the Global domain, so I think it's fine to leave it.